PythonAPI/docs/world.yml

```diff
@@ -0,0 +1,414 @@
+---
+- module_name: carla
+  doc: >
+  # - CLASSES ------------------------------
+  classes:
+  - class_name: Timestamp
+    # - DESCRIPTION ------------------------
+    doc: >
+    # - PROPERTIES -------------------------
+    instance_variables:
+    - var_name: frame
+      type: int
+      doc: >
+        The number of frames elapsed since the simulator was launched.
+    - var_name: elapsed_seconds
+      type: float
+      doc: >
+        Simulated seconds elapsed since the beginning of the current episode.
+    - var_name: delta_seconds
+      type: float
+      doc: >
+        Simulated seconds elapsed since the previous frame.
+    - var_name: platform_timestamp
+      type: float
+      doc: >
+        Time-stamp of the frame at which this measurement was taken, in seconds
+        as given by the OS.
+    # - METHODS ----------------------------
+    methods:
+    - def_name: __init__
+      params:
+      - param_name: frame
+        type: int
+      - param_name: elapsed_seconds
+        type: float
+      - param_name: delta_seconds
+        type: float
+      - param_name: platform_timestamp
+        type: float
+      doc: >
+    # --------------------------------------
+    - def_name: __eq__
+      params:
+      - param_name: other
+        type: carla.Timestamp
+      doc: >
+    # --------------------------------------
+    - def_name: __ne__
+      params:
+      - param_name: other
+        type: carla.Timestamp
+      doc: >
+    # --------------------------------------
+    - def_name: __str__
+      doc: >
+    # --------------------------------------
+
+  - class_name: ActorList
+    # - DESCRIPTION ------------------------
+    doc: >
+      Class that provides access to actors.
+    # - METHODS ----------------------------
+    methods:
+    - def_name: find
+      params:
+      - param_name: actor_id
+        type: int
+      doc: >
+    # --------------------------------------
+    - def_name: filter
+      params:
+      - param_name: wildcard_pattern
+        type: str
+      doc: >
+    # --------------------------------------
+    - def_name: __getitem__
+      params:
+      - param_name: pos
+      doc: >
+    # --------------------------------------
+    - def_name: __len__
+      doc: >
+    # --------------------------------------
+    - def_name: __iter__
+      doc: >
+    # --------------------------------------
+    - def_name: __str__
+      doc: >
+    # --------------------------------------
+
+  - class_name: WorldSettings
+    # - DESCRIPTION ------------------------
+    doc: >
```

Comment:
Here a link to https://carla.readthedocs.io/en/latest/configuring_the_simulation/ would be nice